backend/onyx/connectors/google_drive/doc_conversion.py

```diff
@@ -131,21 +131,28 @@ def _extract_sections_basic(
                     tmp.write(response)
                     tmp_path = tmp.name
 
-                xls = pd.ExcelFile(tmp_path)
+                TEXT_SECTION_SEPARATOR = "\n\n"
+
+                workbook = openpyxl.load_workbook(tmp_path, read_only=True)
```

Comment:
Emulated  file connector excel file read implementation [here](https://github.com/onyx-dot-app/onyx/blob/1f2af373e1236f15058ebb5f6dd10e6551cca10e/backend/onyx/file_processing/extract_file_text.py#L304).
notes:
I did not just import this function because the ramifications of cramming all the sheet tabs into one Section instance was ambiguous. Wanted to stick to the original return type of the function so the less likely code would break.